virtual_rainforest/models/animals/animal_model.py

```diff
@@ -110,11 +113,43 @@ def spinup(self) -> None:
         """Placeholder function to spin up the animal model."""
 
     def update(self, time_index: int) -> None:
-        """Placeholder function to solve the animal model.
+        """Function to step the animal model through time.
+
+        Currently this is a toy implementation.
```

Comment:
Even being a toy implementation, the implicit call of methods I point out seems odd - and unnecessary.

Reply:
My thinking was `apply_community_method` as a placeholder for what will be a variety of community-method specific model-level handlers. It's absolutely not necessary right now but the implicit approach may become necessary once I am implementing multi-grid occupancy and issues like "how does dispersal work?" become more complicated. If it's a problem in the interim I can always simplify right now and rebuild the specific ones I need down the road.